### ISSUE TITLE
Provide skipBatchPath also in the Web Interface

### ISSUE THREAD

**Issue by gregorwolf** [WillEastbury/odata-openapi#5]
**Is your feature request related to a problem? Please describe.**

Right now in the Optional Parameters at https://convert.odata-openapi.net/ it's not possible to configure skipBatchPath.

**Describe the solution you'd like**

The CLI Tool provides the parameter [skipBatchPath](https://github.com/oasis-tcs/odata-openapi/blob/main/lib/README.md?plain=1#L66). Would be great if the Web Interface could follow that example.

**Describe alternatives you've considered**

Using the CLI Tool instead of the Web Interface

**Comment by MartinPankraz** [comment]
Good idea @gregorwolf. In our SPA we are using the XSL files provided, Any suggestion how to inject that change? Here is the [code section](https://github.com/MartinPankraz/odata-openapi/blob/main/index.html#L256).
